index.js

```diff
@@ -12,7 +15,11 @@ function getAssignmentTypes(assignments) {}
  * @param {number} score.max - The maximum amount of points that could be received on the assignment.
  * @returns {number} A rounded integer, representing a percentage.
  */
-function getPercentageScore(score) {}
+function getPercentageScore(score) {
+   let percent = (score.received / score.max) * 100
+   return Math.round(percent)
```

Comment:
You don't _need_ the percent variable. Though you're _certainly_ welcome to prefer breaking it down in the end, try making this all one line for an exercise in concise coding. It's a good exercise in different styles of coding!